lib/Infer/InstSynthesis.cpp

```diff
@@ -1469,8 +1469,9 @@ void findCands(Inst *Root, std::vector<Inst *> &Guesses, InstContext &IC,
     ++Benefit;
     if (Visited.insert(I).second) {
       if (I->K != Inst::Phi) {
-        for (auto Op : I->Ops)
+        for (auto Op : I->Ops) {
```

Comment:
I think we should choose one of using parens for single-line loops/conditionals or not using them, and try to be consistent, but either way I don't think you should change this in existing code except in a cleanup patch

Reply:
Sure.